app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
 
   validates_presence_of :first_name, :last_name
 
+  def full_name
+    "#{first_name} #{last_name}"
+  end
+
```

Comment:
@RobertCram @weedySeaDragon ok I will delete `#full_name` in lines `10-12` in `User`